lesson001/task1/lesson1-1.html

```diff
@@ -7,15 +7,15 @@
     <body>
         <h1>HTML</h1>
         <p>
-            Язык <b>HTML</b> интерпретируется <u>браузерами</u>; полученный в результате интерпретации <br />
+            Язык <b>HTML</b> интерпретируется <a href="browser.html"><u>браузерами</u></a>; полученный в результате интерпретации <br />
             форматированный текст отображается на экране монитора компьютера или мобильного устройства. <br />
```

Comment:
)))) Here must be another link. If you click on the link in the original document, you will be taken to the Wikipedia article. and in file with logic markup must be links too.

Reply:
Completed